assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -169,11 +169,18 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
     !snapDisabled && (!isDragging || (isDragging && !activeDropTargetId));
   const hideHandles = (isDragging && isMaskable) || Boolean(draggingResource);
 
+  const minWidth = dataToEditorY(resizeRules.minWidth);
+  const minHeight = dataToEditorY(resizeRules.minHeight);
+  const aspectRatio = selectedElement.width / selectedElement.height;
+
+  const visuallyHideHandles =
+    selectedElement.width <= minWidth || selectedElement.height <= minHeight;
```

Comment:
This doesn't work on smaller screens - because e.g. `minWidth` is in editor pixels and `selectedElement.width` is in data pixels.

The following seems to work at all screen sizes by using the "raw" values in data pixels:

```suggestion
  const visuallyHideHandles =
    selectedElement.width <= resizeRules.minWidth ||
    selectedElement.height <= resizeRules.minHeight;
```

Reply:
Right, I didn't check this part on a small canvas.